sessionctx/variable/session.go

```diff
@@ -826,6 +826,10 @@ type SessionVars struct {
 
 	// EnableDynamicPrivileges indicates whether to permit experimental support for MySQL 8.0 compatible dynamic privileges.
 	EnableDynamicPrivileges bool
+
+	// CteMaxRecursionDepth indicates The common table expression (CTE) maximum recursion depth.
+	// see https://dev.mysql.com/doc/refman/8.0/en/server-system-variables.html#sysvar_cte_max_recursion_depth
+	CteMaxRecursionDepth int
```

Comment:
Will it be better to name it `CTEMaxRecursionDepth`?

Reply:
PTAL